resources/js/BaseComponents/Navbar.vue

```diff
@@ -4,14 +4,31 @@
     <input class="form-control form-control-dark w-100" type="text" placeholder="Search" aria-label="Search">
     <ul class="navbar-nav px-3">
       <li class="nav-item text-nowrap">
-        <a class="nav-link" href="#">Sign out</a>
+        <a class="nav-link" 
+           href="#"
+           @click="logOut">
+          Sign out
+        </a>
       </li>
     </ul>
   </nav>
 </template>
 
 <script>
+
+import { LOGOUT } from "../store/actions.type";
+
 export default {
-    
+    mounted(){
+
+    },
+    methods:{
+      logOut(){
```

Comment:
add space lang ta before `{` sir
`
logOut() {
  this.$store.dispatch(LOGOUT)
  .then(() => {
    this.$router.push('/login')
  })
}
`